web/thesauruses/clojure/1/operators.json

```diff
@@ -0,0 +1,163 @@
+    "language_name": "Clojure",
+    "structure": "operators"
+  },
+  "concepts": {
+    "addition": {
+      "code": "(+ 1 2 3)",
+      "name": "Addition operator"
+    },
+    "addition_assignment": {
+      "comment": "Clojure doesn't have the traditional notion of assignment as in procedural languages, but it can apply in some situations (ex: atoms and volatile refs).",
+      "code": "(let [x (volatile! 0)]\n  (doseq [addend (range 10)]\n    (vswap! x + addend))\n  @x)",
+      "name": "Addition and assignment operator"
+    },
+    "subtraction": {
+      "code": "(- 10 4)",
+      "name": "Subtraction operator"
+    },
+    "subtraction_assignment": {
+      "comment": "See note about the Addition and assignment operator above.",
+      "not-implemented": "true",
+      "name": "Subtraction and assignment operator"
+    },
+    "multiplication": {
+      "code": "(* 12 12)",
+      "name": "Multiplication operator"
+    },
+    "multiplication_assignment": {
+      "comment": "See note about the Addition and assignment operator above.",
+      "not-implemented": "true",
+      "name": "Multiplication and assignment operator"
+    },
+    "division": {
+      "code": "(/ 1024 16)",
+      "name": "Division operator"
+    },
+    "division_assignment": {
+      "comment": "See note about the Addition and assignment operator above.",
+      "not-implemented": "true",
+      "name": "Division and assignment operator"
+    },
+    "modulus": {
+      "code": "(mod 18 8)",
+      "name": "Modulus (remainder) operator"
+    },
+    "modulus_assignment": {
+      "comment": "See note about the Addition and assignment operator above.",
+      "not-implemented": "true",
+      "name": "Modulus and assignment operator"
+    },
+    "increment": {
+      "comment": "See note about the Addition and assignment operator above. The `inc` function will increment the value and return the result, not overwrite any \"variable\".",
+      "code": "(inc 1)",
+      "name": "Increment (add 1) operator"
+    },
+    "decrement": {
+      "comment": "See note about the Addition and assignment operator above. The `dec` function will decrement the value and return the result, not overwrite any \"variable\".",
+      "code": "(dec 10)",
+      "name": "Decrement (subtract 1) operator"
+    },
+    "exponential": {
+      "comment": "No built in function, but easy enough to write.",
+      "code": "(defn exp [x n]\n  (reduce * (repeat n x)))",
+      "name": "Exponential operator"
+    },
+    "factorial": {
+      "comment": "No built in function, but easy enough to write.",
+      "code": "(defn ! [n]\n  (reduce *' (range 1 (inc n))))",
+      "name": "Factorial operator"
+    },
+    "absolute_value": {
+      "comment": "Using the built-in JDK method.",
+      "code": "(Math/abs -4)",
+      "name": "Absolute value operator"
+    },
+    "percentage": {
+      "code": "%",
+      "name": "Percentage operator",
+      "not-implemented": "true"
```

Comment:
Is this % or not implemented? Can't have both.

Reply:
Sorry, in this case I actually did check the rendered website (which was correct) but forgot to remove it from source).  Should be fixed now.